src/parser/shared/modules/items/Healthstone.js

```diff
@@ -1,18 +1,37 @@
 import SPELLS from 'common/SPELLS/index';
 
+import Combatants from 'parser/shared/modules/Combatants';
+import Abilities from 'parser/core/modules/Abilities';
+import Buffs from 'parser/core/modules/Buffs';
+import SpellUsable from 'parser/shared/modules/SpellUsable';
+import AbilityTracker from 'parser/shared/modules/AbilityTracker';
+import { Class } from 'parser/core/Events';
 import Potion from './Potion';
-
 /**
  * Tracks Healthstone cooldown.
  */
 class Healthstone extends Potion {
+  // Potion does not use Combatants, other dependencies are inherit from Potion.
+  static dependencies = {
```

Comment:
I believe the preferred way to do this is


``` 
dependencies = {
    ...Potion.dependencies,
    combatants: Combatants
 }

```

Which saves you from having to import unused deps

Reply:
Oh right, I completely forgot about that, will change it once I've got time for this!